localstack/services/s3/models.py

```diff
@@ -48,6 +49,10 @@ class S3Store(BaseStore):
         default=dict
     )
 
+    bucket_analytics_configuration: Dict[
+        BucketName, AnalyticsConfigurationList
+    ] = CrossRegionAttribute(default=dict)
+
```

Comment:
nit: we've seen this pattern emerge several times, and I personally prefer to use dictionaries when storing this kind of data. It makes it much easier to access a specific element and also delete them. When calling `List*` operations, you can simply uses `.values()`, and it keeps the order of insertion. I'd be in favour of changing this to `Dict[BucketName, Dict[AnalyticsId, AnalyticsConfiguration]]` instead.

Reply:
Okay. Thanks for the suggestion. I used list earlier to keep it in sync with the inbuilt type that ASF provides. But i agree it will be a lot easier and clean to use dict.